flink-python/pyflink/fn_execution/beam/beam_operations_fast.pyx

```diff
@@ -192,6 +194,108 @@ cdef class PandasAggregateFunctionOperation(BeamStatelessFunctionOperation):
         return generate_func, user_defined_funcs
 
 
+cdef class PandasBatchOverWindowAggregateFunctionOperation(BeamStatelessFunctionOperation):
```

Comment:
It seems to me that most code of beam_operations_fast and beam_operation_slow are the same. Is it possible to avoid the duplication? 

PS: This is not an issue introduced in this PR. We could also address it in a separate if it's possible.

Reply:
I think it is necessary to re-layer the existing operations to be able to achieve only one python implementation at the bottom layer or cython implementation at the same time. I suggest to solve this problem separately in other PRs. What do you think?

Reply:
It makes sense to create a separate PR for this purpose. Could you create a JIRA to track it?

Reply:
Of course. I have created a JIRA https://issues.apache.org/jira/browse/FLINK-19412 to track it.